onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_lib/NeuralNetworksWrapper.h

```diff
@@ -99,12 +102,40 @@ inline std::string TypeToStr(const Type& type) {
   }
 }
 
+struct SymmPerChannelQuantParams {
+  ANeuralNetworksSymmPerChannelQuantParams params;
+  std::vector<float> scales;
+  SymmPerChannelQuantParams(std::vector<float> scalesVec, uint32_t channelDim)
```

Comment:
>std::vector<float> [](start = 28, length = 18)

&&？

Reply:
For this one I would prefer a copy, in my code I was using this vector after creating the SymmPerChannelQuantParams